dev-vcs/bfg-bin/bfg-bin-1.13.0.ebuild

```diff
@@ -1,24 +1,25 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
 inherit java-pkg-2
 
+MY_PN=${PN/-bin/}
+
 DESCRIPTION="A faster alternative to git-filter-branch for removing bad data from git repos"
 HOMEPAGE="https://rtyley.github.io/bfg-repo-cleaner/"
-SRC_URI="https://repo1.maven.org/maven2/com/madgag/${PN}/${PV}/${P}.jar"
+SRC_URI="https://repo1.maven.org/maven2/com/madgag/${MY_PN}/${PV}/${MY_PN}-${PV}.jar"
```

Comment:
@fkobi why `${MY_PN}` instead of simply `bfg`?  see https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Plaintexting_.24.7BPN.7D_in_SRC_URI